### ISSUE TITLE
My Site: Help VoiceOver identify quick access buttons 

### ISSUE THREAD

**Issue by ctarda** [wordpress-mobile/WordPress-iOS#15000]
The buttons in the header of the My Site screen (highlighted in the screenshot below) provide the same functionality than some of the table view cells below them. 

Therefore we have a few buttons (Stats, Pages, Posts, Media) than are actually duplicated, being present in the header and also in the table below it. 

This is confusing when navigating this screen using VoiceOver, as there is no explanation of why those buttons are duplicated.

A potential solution, suggested by a user, is to mark these buttons as "Quick Access", either by grouping them under a separate tab or by a dedicated header.

Screenshot: quick access buttons
<Img src="https://user-images.githubusercontent.com/2722505/94389023-3d8d2000-0181-11eb-8ad9-80b23a048d0c.jpeg" width="350" alt="Screenshot of My Sites highlighting the Quick Access buttons"/>

### Expected behavior
- The four buttons in the header should be marked as a separate section, that indicates that these are sort of quick access shortcuts to some of the features available in this screen

### Actual behavior
- The buttons are exposed to VoiceOver but there is no way to differentiate them from their counterparts in the table view that actually provide the same functionality, and therefore these are perceived as duplicated

### Steps to reproduce the behavior
- Activate VoiceOver
- Navigate to My Site
- Navigate down the view. The best way to reproduce the behaviour is by navigating the view structure by swiping right to move the VoiceOver cursor from one control to another
- Continue navigating past the first four buttons. Notice how, eventually, there will more controls read as Stats, Pages...

##### Tested on [device], iOS [version], WPiOS [version]
Tested on iPhone 6s+, iOS 13.7, WPiOS 15.7, es_ES
Tested on iPhone 11 Pro Max, iOS 14.0.1, WPiOS 15.7, en_CA

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
